pkg/apis/compliance/v1alpha1/tailoredprofile_types.go

```diff
@@ -6,6 +6,18 @@ import (
 
 // FIXME: move name/rationale to a common struct with an interface?
 
+// OutdatedReferenceValidationDisable a label is used to disable validation of outdated references
+const OutdatedReferenceValidationDisable = "compliance.openshift.io/outdated-reference-validation-disable"
+
+// RuleLastCheckTypeChangedAnnotationKey is the annotation key used to indicate that the rule check type has changed, store its previous check type
+const RuleLastCheckTypeChangedAnnotationKey = "compliance.openshift.io/rule-last-check-type"
+
+// VariableDeprecatedAnnotationKey is the annotation key used to indicate that the variable is deprecated
+const VariableDeprecatedAnnotationKey = "compliance.openshift.io/variable-deprecated"
+
+// RulesDeprecatedAnnotationKey is the annotation key used to indicate that the rule is deprecated
+const RulesDeprecatedAnnotationKey = "compliance.openshift.io/rule-deprecated"
```

Comment:
nit: for readability I would consider putting the verb first (`DeprecatedRuleAnnotationKey`, `compliance.openshift.io/deprecated-rule`)

For the key, do we need to include `rule` if the annotation is always used on `Rules`? Is it implied based on what we use it with? Additionally, do we want to keep this open for use elsewhere besides `Rule` CRDs?

Reply:
that's a good question, I think we get `compliance.openshift.io/deprecated-rule` annotation from the content during the parsing. I am not sure I understand the last question wondering if you could give an example.